### DISCUSSION TITLE
[Bug] 'Public Sponsor' achievement doesn't show on profile

### DISCUSSION THREAD

**Discussion by zaneshaw** [community/community#35284] [Other Feature Feedback, Questions, & Ideas] [answered]
I have an issue where the public sponsor achievement shows correctly on my ['achievements' tab](https://github.com/zaneshaw?tab=achievements), but not on my [profile page](https://github.com/zaneshaw). Is this intended or a bug?

**Comment by radar-blip** [comment, marked as answer]
That does seem a little odd. 
Could be a bug. I would wait a day or two, if it doesn't show up, contact support about it.

**Reply by zaneshaw** [reply]
It's already been occuring for about a fortnight. I think I'll contact them now. Was just wondering if anyone else was experiencing this issue.

**Reply by radar-blip** [reply]
I've never heard of this happening, or read about it anywhere.
If its been that long, its time to call and claim your prize! 
Good luck!! 👍🏻

**Comment by 0einstein0** [comment]
My highlights also disappeared after sponsoring, I guess there is some bug

**Comment by jfullstackdev** [comment]
it's there already, I'm using Incognito, I'm not even logged in

![image](https://user-images.githubusercontent.com/47092464/194710230-cef78c63-ad40-4937-a619-f85ad671f17c.png)

it's maybe a cached data on your end or maybe extensions preventing it to be displayed

**Reply by zaneshaw** [reply]
Sorry, this was resolved by GitHub support yesterday.
